openapi.yaml

```diff
@@ -57,11 +57,17 @@ paths:
       tags:
         - games
       operationId: getGames
-      parameters: 
+      parameters:
         - $ref: '#/parameters/authorization'
         - in: query
           name: round
           type: string
+          enum:
+            - blind
+            - flop
+            - turn
+            - river
+            - showdown
```

Comment:
Could you use a YAML anchor for the enum like we discussed so you don't need to repeat the code here? https://swagger.io/docs/specification/2-0/enums/